src/transactions/feed/TransactionFeed.tsx

```diff
@@ -43,7 +48,7 @@ function TransactionFeed() {
     useFetchTransactions()
 
   const cachedTransactions = useSelector(transactionsSelector)
-
+  Logger.info(TAG, cachedTransactions)
```

Comment:
did you mean to leave this here? should it be Logger.debug or removed?

Reply:
Oops, should be removed